### ISSUE TITLE
Stackable 2: save custom blocks to "Designs" section

### ISSUE THREAD

**Issue by nicmare** [gambitph/Stackable#500]
You provide predefined designs in each block. As far as i can see this are just some kind of block templates with modified block settings. Would be nice to add my own Design by code.  
Are there any plans?

**Comment by bfintal** [comment]
Yes there are plans to have an API so that people can add in their own designs. Although right now the designs area isn't yet complete and just resides in the inspector, the plan it to enable it to expand to display more designs. We'll do a designs API then :)

**Comment by bfintal** [comment]
Closing as superseded. Custom design saving is now handled by:

- **Global Block Styles** (v3.18) — save and reuse block styles across your site: https://wpstackable.com/blog/introducing-block-styles/
- The old Block Defaults feature was sunset in favor of the Design System (#3500)

Saving custom patterns & pages to the Design Library is on our [roadmap](https://wpstackable.com/roadmap) (item #7). For programmatic customization, filter hooks remain available in the codebase.
